### ISSUE TITLE
Move the documentation for event size limits to the room version spec

### ISSUE THREAD

**Issue by anoadragon453** [matrix-org/matrix-spec#1208]
Currently it lives [in the Client-Server API](https://spec.matrix.org/v1.3/client-server-api/#size-limits), but will likely be tied to a room version if changed. Thus it should live in the room version definitions.

**Comment by anoadragon453** [comment]
@turt2live says that we should only do this once it differs between room versions, otherwise it will cause confusion.

**Comment by richvdh** [comment]
Yeah, I take the point, but the problem with it not being in the room versions doc is that I'm not sure where it *should* be. It's not really specific to either the C-S API or the S-S API. It really lives alongside the ["Event format"](https://spec.matrix.org/v1.3/rooms/v1/#event-format) imho.

**Comment by turt2live** [comment]
It would live in the appendices if we didn't have room versions, so there seems like a reasonable home if we don't plan on making room version-needed changes to the size.

**Comment by richvdh** [comment]
I guess that's fair, provided we link to it from other places.
